src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -150,6 +162,7 @@
     <Compile Include="**\*.wpf.cs" />
     <Compile Include="**\*.wpf.*.cs" />
     <PackageReference Include="Xamarin.Forms.Platform.WPF" Version="5.0.0.1709-pre4" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
```

Comment:
Not sure if we need this here since it's under the .net framework ItemGroup as well.

Reply:
I'm not sure how this got in there either. I will take a look and see what I can find

Reply:
I have removed this for the next code push